fiftyone/core/labels.py

```
@@ -2103,3 +2103,218 @@ def _from_geo_json(d):
+            ImportError: if required dependencies (trimesh, numpy) are not available
+        """
+        import hashlib
+        import os
+
+        try:
+            import numpy as np
+        except ImportError:
+            raise ImportError(
+                "numpy is required for exporting HumanPose3D scenes. "
+                "Install it with: pip install numpy"
+            )
```

Ruff (0.14.5)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)